Realm/ObjectServerTests/RLMObjectServerTests.mm

```diff
@@ -2002,64 +1998,65 @@ - (void)testMongoFind {
 }
 
-- (void)testMongoAggregateAndCount {
-    RLMMongoClient *client = [self.anonymousUser mongoClientWithServiceName:@"mongodb1"];
-    RLMMongoDatabase *database = [client databaseWithName:@"test_data"];
-    RLMMongoCollection *collection = [database collectionWithName:@"Dog"];
-
-    [self cleanupRemoteDocuments:collection];
-
-    XCTestExpectation *insertManyExpectation = [self expectationWithDescription:@"should insert one document"];
-    [collection insertManyDocuments:@[
-        @{@"name": @"fido", @"breed": @"cane corso"},
-        @{@"name": @"fido", @"breed": @"cane corso"},
-        @{@"name": @"rex", @"breed": @"tibetan mastiff"}]
-                         completion:^(NSArray<RLMObjectId *> * objectIds, NSError * error) {
-        XCTAssertTrue(objectIds.count == 3);
-        XCTAssertNil(error);
-        [insertManyExpectation fulfill];
-    }];
-    [self waitForExpectationsWithTimeout:60.0 handler:nil];
-
-    XCTestExpectation *aggregateExpectation1 = [self expectationWithDescription:@"should aggregate documents"];
-    [collection aggregateWithPipeline:@[@{@"name" : @"fido"}]
-                           completion:^(NSArray<NSDictionary *> * documents, NSError * error) {
-        XCTAssertNotNil(error);
-        XCTAssertTrue([error.domain.description isEqualToString:@"realm::app::ServiceError"]);
-        XCTAssertNil(documents);
-        [aggregateExpectation1 fulfill];
-    }];
-    [self waitForExpectationsWithTimeout:60.0 handler:nil];
-
-    XCTestExpectation *aggregateExpectation2 = [self expectationWithDescription:@"should aggregate documents"];
-    [collection aggregateWithPipeline:@[@{@"$match" : @{@"name" : @"fido"}}, @{@"$group" : @{@"_id" : @"$name"}}]
-                           completion:^(NSArray<NSDictionary *> * documents, NSError * error) {
-        XCTAssertNil(error);
-        XCTAssertNotNil(documents);
-        XCTAssertTrue(documents.count > 0);
-        [aggregateExpectation2 fulfill];
-    }];
-    [self waitForExpectationsWithTimeout:60.0 handler:nil];
-
-    XCTestExpectation *countExpectation1 = [self expectationWithDescription:@"should aggregate documents"];
-    [collection countWhere:@{@"name" : @"fido"}
-                completion:^(NSInteger count, NSError * error) {
-        XCTAssertTrue(count > 0);
-        XCTAssertNil(error);
-        [countExpectation1 fulfill];
-    }];
-    [self waitForExpectationsWithTimeout:60.0 handler:nil];
-
-    XCTestExpectation *countExpectation2 = [self expectationWithDescription:@"should aggregate documents"];
-    [collection countWhere:@{@"name" : @"fido"}
-                     limit:1
-                completion:^(NSInteger count, NSError * error) {
-        XCTAssertEqual(count, 1);
-        XCTAssertNil(error);
-        [countExpectation2 fulfill];
-    }];
-    [self waitForExpectationsWithTimeout:60.0 handler:nil];
-}
+/// FIXME: Re-enable once we understand why the server is not setup correctly
```

Comment:
Temporarily disabling tests should be done by renaming the test rather than commenting it out so that it still gets compiled and doesn't bitrot if we rename things.

When things should be commented out temporarily it should be done with block comments rather than line comments so that it doesn't make the diff and history a mess.